### DISCUSSION TITLE
[Feature] Add option to filter "google sign-in prompt"

### DISCUSSION THREAD

**Discussion by hprodh** [uBlockOrigin/uAssets#23165] [Ideas] [answered]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.perplexity.ai/
```

### Description

For example, on the URL : perplexity.ai, I like to be able to directly type in there after opening on a new tab, but the sign-in prompt is an annoyance as it breaks the keyboard flow.

I know much users actually do not consider google as an annoyance so it could be an option.

The workaround is easy : I just had to add this line to the filters :
accounts.google.com/gsi/*$xhr,script,3p

Yet I wish I could just click a dedicated checkbox to activate this filter just as a default one.

Thank for your attention.

### Other extensions used

none

### Configuration

** no real bug implied

**Comment by Oqarshi** [comment]
I think this is solved using one of the annoyance filters. I could be wrong though

**Reply by krystian3w** [reply, marked as answer]
~~Not in added to recommended.~~ (I see block and hide in "AdGuard – Pop-ups" -- btw removed from uBo Lite).

**Reply by garry-ut99** [reply]
> hprodh : I know much users actually do not consider google as an annoyance so it could be an option.
> Oqarshi : I think this is solved using one of the annoyance filters. I could be wrong though

They say that such sign-in prompts are covered by annoyances filters: 
- https://github.com/uBlockOrigin/uAssets/discussions/14746

<hr>

> hprodh : Yet I wish I could just click a dedicated checkbox to activate this filter just as a default one.

Similiar to:
- https://github.com/uBlockOrigin/uBlock-issues/issues/3224#issuecomment-2077339953 which is a duplicate of
- https://github.com/gorhill/uBlock/issues/972

the difference is that your request is only for google sign-in prompt and not for every filter.

**Comment by hprodh** [comment]
It is actually covered by the Adguard annoyance filter. Problem solved. 
Thank you.
